packages/menu/src/Menu.spec.tsx

```diff
@@ -115,6 +120,27 @@ describe('packages/menu', () => {
         expect(menu).toBeInTheDocument();
       });
     });
+
+    test('Click handlers on parent elements still fire', () => {
```

Comment:
Small nit - `still fire` makes sense in the context of this PR since we're adding `stopPropagation`, but it might be clearer in the future if it just says `fire`